README.md

```diff
@@ -14,6 +14,9 @@ This sample implements a function triggered by Azure Blob Storage to resize an i
 
 The key aspects of this sample are in the function bindings and implementation.
 
+## Use latest Storage Blob SDK

```

Comment:
I have noticed that this is how we are bringing attention to the new package in all applicable sample repos i.e. create a new section with a header etc.

Can we instead have this information in the introduction part of the readme file right under the H1 instead? We can use the `>` symbol and make it appear as a "note"/"quotation"

> IMPORTANT! This sample uses the older version of the Storage Blob SDK i.e. `azure-storage`. Please refer to the sample [storage-blob-resize-function](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob/samples/storage-blob-resize-function) that uses the latest version of the Storage Blob SDK i.e. [@azure/storage-blob](https://www.npmjs.com/package/@azure/storage-blob) instead.

@jongio Thoughts?

Reply:
Or do we want to achieve the above via the repo archival message?

Reply:
Sounds good to me.  @tzhanl got this?

Reply:
@jongio @ramya-rao-a , Now updated with your suggestions.